### ISSUE TITLE
Web root directory - /var/www/html

### ISSUE THREAD

**Issue by sfrancis2017** [DanielnetoDotCom/YouPHPTube#2]
Hi Daniel,

Great work. I tried to move the YouPHPTube to web root directory in Ubuntu, it does not let me make that move, please let me know what config changes need to be made in order to facilitate such a change.

2. As you mentioned, the video is playing in a vertical format. Do you plan to make changes to that?

3. The dropdown menu for flags, I had renamed them and only wanted US and FR to show up - meaning English & French. Where do you set the order by menu to show only the required?

Great work overall. Looking forward to hearing from you.

Sajiv Francis

**Comment by DanielnetoDotCom** [comment]
Hi nice to hear about you
I am away from computer now
If you want add me on WhatsApp +55 71 99346-3337
And Skype danielbeto.com
I will be able to help you configure your server

Em 27 de abr de 2017 2:54 PM, "sfrancis2017" <notifications@github.com>
escreveu:

Hi Daniel,

Great work. I tried to move the YouPHPTube to web root directory in Ubuntu,
it does not let me make that move, please let me know what config changes
need to be made in order to facilitate such a change.

   1.

   As you mentioned, the video is playing in a vertical format. Do you plan
   to make changes to that?
   2.

   The dropdown menu for flags, I had renamed them and only wanted US and
   FR to show up - meaning English & French. Where do you set the order by
   menu to show only the required?

Great work overall. Looking forward to hearing from you.

Sajiv Francis

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/DanielnetoDotCom/YouPHPTube/issues/2>, or mute the
thread

**Comment by DanielnetoDotCom** [comment]
By the way I am working on vertical videos .
And my Skype was wrong . my Skype is danielneto.com

Em 27 de abr de 2017 17:38, "Daniel Neto" <danielneto.com@gmail.com>
escreveu:

**Comment by sfrancis2017** [comment]
Hi Daniel,

I will add you up on whats app and skype for sure. You have put together a
great software for sharing videos.

Looking forward to collaborating with you. Are you from Brazil? I had
someone from Quebec view those videos and thought it was you.

Best regards

Sajiv

**Comment by DanielnetoDotCom** [comment]
Hi Sajiv

Thanks for Using YouPHPTube

# Question 1
Always when you move YouPHPTube you need to take a look on  `videos/configuration.php` file.
There are 2 variables you need to change.
```
$global['webSiteRootURL'] = 'http://YourDomainToYouPHPTube/';
$global['systemRootPath'] = '/PathTo/YouPHPTube/';
```
# Question 2
It is released in version 2.2 (Just Done 👍 )

# Question 3
Always when you want to add or remove an language, just take a look on the locale/ directory.
You only need the files: 
- functions.php
- index.php
- save.php

The rest are language files you can remove from your system. just remove the file it will disappear

**Comment by sfrancis2017** [comment]
Hi Daniel,

Thanks for the quick response. I will work on creating a documentation on the changes we made.

1. Thank you for the help in setting up the virtual host in Apache for my domain; solves the sub-folder issue. Process: change the root directory by cloning the 000-default.conf to your domain.conf- path in Ubuntu '/etc/apache2/sites-available/default'

2. I will also include the encryption process that I used on my domain in the documentation - Letsencrypt

3. About the language -flag icon changes (Canada), I will look into it this weekend and let you know if there are any hiccups.

Awesome working with you, great speed.
